graylog2-server/src/test/java/org/graylog2/system/shutdown/GracefulShutdownServiceTest.java

```diff
@@ -142,6 +143,7 @@ public void failRegisterAndUnregisterDuringShutdown() {
                 .isInstanceOf(IllegalStateException.class);
     }
 
+    @Disabled("Disabled because it is flaky on Jenkins. Could maybe be fixed by adjusting timeouts.")
```

Comment:
How about using a JUnit environment variable condition to let the test run outside of Jenkins?

```
@DisabledIfEnvironmentVariable(named = "CI", matches = "true", disabledReason = "Flaky on Jenkins")
```

Reply:
Good idea. Thank you!